### ISSUE TITLE
Removing a class attribute results in a "null" class

### ISSUE THREAD

**Issue by AngusMorton** [marko-js/marko#2175]
Using the Tags API Preview, dynamically removing a class attribute results in a class of "null" instead of the class attribute being removed.

Given:
```marko
<const/hidden = true />
<div class=hidden ? "hidden" : "" />
```

When `hidden` is toggled, the `class` attribute will become `"null"` instead of being removed.

See: [Playground Link](https://markojs.com/playground/#NobwRAdghgtgpmAXGAlhAJnAHgOhlAJwGsB7MAGjAAcoAXACyTAHoBjEmKkiOCWgZ2ZpMufMTKV2fXrSbNmAAgDKtOFX6IAOhHkKAjDgUAVenAXoUANwDk-BawA2Ufnbq0CKAEYBXVQpR2mmD0KOiYEEE42roATIYAwg4orEQKDGY+tLTcUTrM0YomZo7OrlkemWYA7igODgqeZkEQ3nWRCgCStAr89CSt6A1mBHAwJJZwgyQEQwpQEAqjVLQAnj3uaADmuQA8DnC0zCFhvAC87t5wzAB82juZ2QvciclEABQAlAogCsfhCqcFABCP68BQAX1uCwUOwslnsThcp1BCwA-AogiiggpEBiwEEoQoiQoACQgFHg7REnbMOFQmkPbjXMDggC6QA)

**Comment by Harmedino** [comment]
would work on this

can i please get more explanation

**Comment by svallory** [comment]
> would work on this
> 
> can i please get more explanation

I checked it out, if you inspect the HTML of the button, you'll see that it becomes `<button class="null">` instead of `<button>`. My guess would be that there's a `JSON.stringify` at play here hehehe

**Comment by Harmedino** [comment]
what the exact button please 
how do i run the app on dev mode no script for dev mode

**Comment by svallory** [comment]
> > > would work on this
> > > can i please get more explanation
> > 
> > 
> > I checked it out, if you inspect the HTML of the button, you'll see that it becomes `<button class="null">` instead of `<button>`. My guess would be that there's a `JSON.stringify` at play here hehehe
> 
> can you please specify the button having issue

Check the playground link in the issue, there's only one button there ;)

**Comment by DylanPiercey** [comment]
@Harmedino apologies I missed your comment, this is resolved after #2178. (Release incoming).
